### DISCUSSION TITLE
fakelottes & fakelottes.tweaked appear to be one and the same

### DISCUSSION THREAD

**Discussion by dros0-0** [dosbox-staging/dosbox-staging#2952] [Q&A] [answered]
Whereas lottes and lottes.tweaked are different, fakelottes and fakelottes.tweaked appear to be the same, albeit with formatting differences inside the glsl files. fakelottes and fakelottes.tweaked appear to have the same effect. I didn't open a bug report, in case there is some key difference I am missing.

https://github.com/dosbox-staging/dosbox-staging/blob/main/contrib/svn-shaders/fakelottes.glsl
https://github.com/dosbox-staging/dosbox-staging/blob/main/contrib/svn-shaders/fakelottes.tweaked.glsl

fakelottes
![fakelottes](https://github.com/dosbox-staging/dosbox-staging/assets/93776065/a8c1d51b-89bf-4864-b635-123e84954cbb)

fakelottes.tweaked
![fakelottes tweaked](https://github.com/dosbox-staging/dosbox-staging/assets/93776065/e137534f-93ee-4384-976c-54ce4e273bf7)

**Comment by johnnovak** [comment, marked as answer]
@dros0-0 All currently bundled CRT shaders will be deprecated and removed in the next release; they will be replaced by the adaptive authethic CRT shaders that emulate PC monitors. More info and screenshots here:

https://github.com/dosbox-staging/dosbox-staging/pull/2777

The current CRT shaders will be available as a separate download and people can continue to use them, but we won't maintain or update them going forward.

For the record, the Lottes variants aim to emulate consumer TVs not PC monitors.
